pygmt/datasets/earth_relief.py

```diff
@@ -120,11 +140,24 @@ def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm
             f"{registration}-registered Earth relief data for "
             f"resolution '{resolution}' is not supported."
         )
-
+    earth_relief_sources = {"igpp": "earth_relief_", "gebco": "earth_gebco_"}
+    if data_source in earth_relief_sources and data_source != "igpp":
+        with Session() as lib:
+            if Version(lib.info["version"]) < Version("6.4.0"):
+                raise GMTVersionError(
+                    f"The {data_source} option is not available for GMT"
+                    " versions before 6.4.0."
+                )
     # Choose earth relief data prefix
-    earth_relief_prefix = "earth_relief_"
-    if use_srtm and resolution in land_only_srtm_resolutions:
+    if use_srtm and resolution in land_only_srtm_resolutions and data_source == "igpp":
         earth_relief_prefix = "srtm_relief_"
+    elif data_source in earth_relief_sources:
+        earth_relief_prefix = earth_relief_sources[data_source]
+    else:
+        raise GMTInvalidInput(
+            f"Invalid earth relief 'data_source' {data_source}, "
+            "valid values are 'igpp' and 'gebco'."
+        )
```

Comment:
Do we need to better handle the case when `use_srtm=True` and `data_source="gebco"`?

Reply:
Fixed to check for `use_srtm=True` and data_source isn't "igpp".